src/stores/video.ts

```diff
@@ -342,76 +461,130 @@ export const useVideoStore = defineStore('video', () => {
   const updateLastProcessingUpdate = (recordingHash: string): void => {
     const info = unprocessedVideos.value[recordingHash]
     info.dateLastProcessignUpdate = new Date()
-    unprocessedVideos.value = { ...unprocessedVideos.value, ...{ [recordingHash]: info } }
+    unprocessedVideos.value = {
+      ...unprocessedVideos.value,
+      ...{ [recordingHash]: info },
+    }
```

Comment:
Is there any change on this line?

Reply:
This was also not manually made. This is an ESLint auto formatting change. 
I will configure my formatting scripts so they will not touch this kind of code.

Reply:
Missing this one.